### ISSUE TITLE
photos with chinese characters don't upload as Main Image

### ISSUE THREAD

**Issue by ebarry** [publiclab/plots2#547]
![17](https://cloud.githubusercontent.com/assets/161439/15610329/af53573e-23f2-11e6-901a-c0a3d1395be6.jpg)

### Relevant URLs

https://publiclab.org/notes/Shizi/05-27-2016/kite-aerial-photography-in-china

### PublicLab.org username

(to help reproduce the issue)

### Browser, version, and operating system

For bug reports, fill out the above template; for feature requests, you can delete the template.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by jywarren** [comment]
Can you post the original name of the image, since it's since been replaced on the linked-to research note? Image uploading uses the `paperclip` gem, and the first step is probably to figure out whether the image:

1. uploaded but does not display, or
2. failed on upload and never got to display. 

I think 1 is probably right, as I saw a "broken" image displayed when this note first published.

**Comment by ebarry** [comment]
ah, i thought uploading it to github would preserve the orignal file name. i guess not. 

图片17.jpg

**Comment by jywarren** [comment]
Hi, Liz, when you have a chance, can you add the steps needed to reproduce
this error? This will make it easier for people to understand and fix the
bug.

**Comment by ananyo2012** [comment]
Yes the image is being uploaded but the name is displayed incorrectly as you can see in this test research note https://publiclab.org/notes/ananyo2012/08-19-2016/testing-out-a-note-within-chinese-characters-in-main-image. This is weird as the same image shows up when you upload it in the body.

**Comment by ananyo2012** [comment]
This is the ASCII format of the image name: `%E5%9B%BE%E7%89%8717.jpg`. if you paste this name beside the image path i.e. https://i.publiclab.org/system/images/photos/000/017/679/original/%E5%9B%BE%E7%89%8717.jpg you will be able to see the uploaded image. But somehow the name is displayed incorrectly in the note. The name of the image is fetched by `@node.main_image.path(:original)` as you can see in this line: https://github.com/publiclab/plots2/blob/master/app/views/notes/show.html.erb#L7

**Comment by ebarry** [comment]
Hi all, just bumping this issue up as image upload -- if the title has non-latin characters -- to the Main Image, as well as Drag-n-Drop into the Body of a research note, is failing: 
<img width="513" alt="screen shot 2018-11-16 at 2 46 43 pm" src="https://user-images.githubusercontent.com/161439/48602562-787f7380-e9ae-11e8-9814-faa124926c1f.png">

**Comment by jywarren** [comment]
I've created two tests -- a functional test and a test fixture -- which should fail, reproducing this error: 

https://github.com/publiclab/plots2/pull/4118

If this PR fails, we would love help resolving it. It might involve database encoding, as in https://github.com/publiclab/plots2/issues/2665
